springerlink.md

```diff
@@ -1,163 +1,1552 @@
+<br><br>
+The Freshdesk customer services widget uses background images without text equivalents to initiate functionality. We are actively working to solve this problem. Additionally, we have provided alternate means of accessing customer services on our "Contact Us" page, including a contact email address, a support ticket system, and helpdesk telephone numbers.
+</td>
+</tr>
+<tr id="use-of-color" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color"><strong>1.4.1 Use of Color</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.10 (Web)</li>
+<li>10.2.10 (non-web document)</li>
+<li>11.2.1.10 (Software)</li>
+<li>11.2.2.10 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Colour alone is generally not used to convey information in the pages of this product.
+<br><br>
+Links and icons may use colour to indicate hover or active states. These are supplemented with visible focus rings and/or background colours and/or text decoration underlines and/or luminosity differences per WAI techniques <a href="https://www.w3.org/WAI/WCAG21/Techniques/general/G183">G183</a> and <a href="https://www.w3.org/WAI/WCAG21/Techniques/css/C15">C15</a>.
+<br><br>
+Some complex images in journals, books, series, protocols, reference works and proceedings may rely on colour as the only visual means of understanding information. Examples may include images of charts, graphs, diagrams or maps. Generally, these images are used to supplement text content. 
+</td>
+</tr>
+<tr id="audio-control" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#visual-audio-contrast-dis-audio"><strong>1.4.2 Audio Control</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.11 (Web)</li>
+<li>10.2.11 (non-web document)</li>
+<li>11.2.1.11 (Software)</li>
+<li>11.2.2.11 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+Content supplied by us does not play automatically. 
+<br><br>
+All video with audio hosted through the third party MovingImage platform has a mechanism for pausing, stopping, and controlling volume. 
+<br><br>
+We do not support the playing of audio only directly on our pages. Audio files hosted by us are supplied as downloadable files only, for use in the users' own media player. 
```

Comment:
should audio only be hyphenated here? it is hyphenated at the beginning of the document